modules/installation-guide/partials/proc_upgrading-the-chectl-management-tool.adoc

```diff
@@ -10,8 +10,8 @@ This section describes how to upgrade the `{prod-cli}` management tool.
 .Prerequisites
 
 *  The `{prod-cli}` tool is available. See:
-** link:#installing-the-{prod-cli}-management-tool-on-windows_{context}[Installing the {prod-cli} management tool on Windows]
-** link:#installing-the-{prod-cli}-management-tool-on-linux-or-macos_{context}[Installing the {prod-cli} management tool on Linux or macOS]
+** xref:using-the-chectl-management-tool.adoc#installing-the-{prod-cli}-management-tool-on-windows_{context}[Installing the {prod-cli} management tool on Windows]
```

Comment:
I see that because this section is upstream only, you replaced `{prod-cli}-management-tool` with `chectl-management-tool` but only in one place. Why not both? it'll always be chectl here, not crwctl. Right?

Or is it that the FILENAME is always chectl, but the link within the page could be either chectl or crwctl?

Reply:
Because install and upgrade procedures are completely different downstream.

Reply:
I misunderstood the question: because we use the attribute in the `id` but not in the file name.